src/xcode/ENA/ENA/Source/Scenes/Home/__tests__/HomeTableViewModelTests.swift

```diff
@@ -51,6 +51,78 @@ class HomeTableViewModelTests: CWATestCase {
 		XCTAssertEqual(viewModel.riskAndTestResultsRows, [.risk])
 	}
 
+	func testRiskCellNotHiddenIfFamilyTestsExistAndUnreadCountIs2() {
```

Comment:
I think it would be nice to also have a test that it is hidden if there are no family tests. Maybe we can replace on of these, as the unread count has no influence on showing the cell

Reply:
changed tests a bit